posthog/settings/sentry.py

```diff
@@ -74,17 +74,24 @@ def traces_sampler(sampling_context: dict) -> float:
 def sentry_init() -> None:
     if not TEST and os.getenv("SENTRY_DSN"):
         sentry_sdk.utils.MAX_STRING_LENGTH = 10_000_000
-        # https://docs.sentry.io/platforms/python/
-        sentry_logging = LoggingIntegration(level=logging.INFO, event_level=None)
+
+        # Setting this on enables move visibility, at the risk of capturing personal information we should not:
+        #   - standard sentry "client IP" field, through send_default_pii
+        #   - django access logs (info level)
+        #   - request payloads
+        send_pii = get_from_env("SENTRY_SEND_PII", type_cast=bool, default=False)
+
+        sentry_logging_level = logging.INFO if send_pii else logging.ERROR
+        sentry_logging = LoggingIntegration(level=sentry_logging_level, event_level=None)
         profiles_sample_rate = get_from_env("SENTRY_PROFILES_SAMPLE_RATE", type_cast=float, default=0.0)
 
         sentry_sdk.init(
             dsn=os.environ["SENTRY_DSN"],
             integrations=[DjangoIntegration(), CeleryIntegration(), RedisIntegration(), sentry_logging],
-            request_bodies="always",
+            request_bodies="always" if send_pii else "never",
             sample_rate=1.0,
             # Configures the sample rate for error events, in the range of 0.0 to 1.0. The default is 1.0 which means that 100% of error events are sent. If set to 0.1 only 10% of error events will be sent. Events are picked randomly.
```

Comment:
I can't figure out if the comment here refers to sample_rate or traces_sample or both, should probably push `send_default_pii` so it's below `request_bodies`